package.json

```diff
@@ -27,6 +27,7 @@
     "lodash.pick": "4.4.0",
     "lodash.set": "4.3.2",
     "selenium-webdriver": "4.1.2",
+    "uiohook-napi": "^1.1.0",
```

Comment:
I didn't get this to work on my Mac and it also seems to fail on the GitHub Action on Mac? I'm thinking two things:
* Maybe we could have it as an optional dependency (and follow the pattern we have for that)? But it should still work on all platforms, do you know if it should? I could also see in the package.json for that package that it seems to be NodeJS 16 and never.
* Lets sync how we should communicate between Browsertime and the browser. If we can't get keys to work, maybe we could use devtools and global variables in JS?

Reply:
Ah good catch! I was running in node-16 so maybe it pulled a version that only works for node-16.